tests/functional/u/use/use_yield_from.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring, import-error
```

Comment:
Could you add 
```suggestion
yield 1
```

Although it is a syntax error I would like to check that `pylint` doesn't crash on it.